23/09/21  12:45 - HTTPConnectionPool(host='172.60.7.105', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1143444634&pass=Control2020*&function=update_campaign&campaign_id=UPSELL1&auto_dial_level=1.5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd30f69d040>: Failed to establish a new connection: [Errno 110] Connection timed out')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 267
23/09/21  12:45 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
23/09/21  12:45 - HTTPConnectionPool(host='172.60.7.105', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1129532518&pass=1129532518&function=update_campaign&campaign_id=PORTABA2&auto_dial_level=1.4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f740ba4f2b0>: Failed to establish a new connection: [Errno 110] Connection timed out')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 267
23/09/21  12:45 - HTTPConnectionPool(host='172.60.7.105', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1129525544&pass=Claro2021&function=update_campaign&campaign_id=CLIENTE1&auto_dial_level=1.4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ae2f612e0>: Failed to establish a new connection: [Errno 110] Connection timed out')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 267
23/09/21  12:45 - HTTPConnectionPool(host='172.60.7.105', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1129525544&pass=Claro2021&function=update_campaign&campaign_id=CLIENTE2&auto_dial_level=1.9 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f20e8c041f0>: Failed to establish a new connection: [Errno 110] Connection timed out')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 267
23/09/21  12:45 - HTTPConnectionPool(host='172.60.7.105', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1129532518&pass=1129532518&function=update_campaign&campaign_id=MIGRACIO&auto_dial_level=1.4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc9d27240a0>: Failed to establish a new connection: [Errno 110] Connection timed out')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 267
23/09/21  12:45 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
23/09/21  12:45 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
23/09/21  12:45 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
23/09/21  12:45 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
23/09/21  12:50 - HTTPConnectionPool(host='172.45.7.21', port=80): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=1007110960&pass=1031840868&function=update_campaign&campaign_id=MVCIERRE&auto_dial_level=1.0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa42754eac0>: Failed to establish a new connection: [Errno 110] Connection timed out')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 267
23/09/21  12:50 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
23/09/21  12:51 - HTTPConnectionPool(host='172.66.7.112', port=39743): Max retries exceeded with url: /vicidial/non_agent_api.php?source=test&user=7318651&pass=7318651&function=update_campaign&campaign_id=FAMISANA&auto_dial_level=1.5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa43442a3d0>: Failed to establish a new connection: [Errno 110] Connection timed out')) - <class 'requests.exceptions.ConnectionError'> - AdminVcdial.py - Line: 255
23/09/21  12:51 - catching classes that do not inherit from BaseException is not allowed - <class 'TypeError'> - AdminVcdial.py - Line: 425
